documentation/projects/proposals/dark_mode/20240313-project_proposal_dark_mode.md

```diff
@@ -0,0 +1,81 @@
+# 2024-03-13 Project Proposal
+
+**Author**: @fcoveram
+
+## Reviewers
+
+<!-- Choose two people at your discretion who make sense to review this based on their existing expertise. Check in to make sure folks aren't currently reviewing more than one other proposal or RFC. -->
+
+- [ ] @obulat
+- [ ] @zackkrida
+
+## Project summary
+
+<!-- A brief one or two sentence summary of the project's features -->
+
+Include a feature that changes the UI from light to a dark color scheme.
+
+### Motivation
+
+Dark mode is a customization level that is in the spirit of adapting the UI to
+people’s contexts. The benefits span a11y, personal preferences set on the
+devices, energy efficiency in devices, and an alternative browsing experience of
+visual content due to the background contrast.
+
+## Goals
+
+<!-- Which yearly goal does this project advance? -->
+
+Refine Search Experience
+
+## Requirements
+
+<!-- Detailed descriptions of the features required for the project. Include user stories if you feel they'd be helpful, but focus on describing a specification for how the feature would work with an eye towards edge cases. -->
+
+1. Design documentation of pages and UI components in the Design Library and any
+   other additional Figma files.
+2. Design documentation of transition from current to new frontend
+   implementation.
+
+## Success
+
+<!-- How do we measure the success of the project? How do we know our ideas worked? -->
+
+- Add the dark theme to the site by allowing visitors to change the UI through
+  three options: Light theme, Dark theme, or follow device setting.
+- Document the dark variant of every UI element in both the Design Library and
+  Storybook.
+
+## Participants and stakeholders
+
+<!-- Who is working on the project and who are the external stakeholders, if any? Consider the lead, implementers, designers, and other stakeholders who have a say in how the project goes. -->
+
+- Lead: @fcoveram
+- Design: @fcoveram
+- Implementation: TBD
+
+## Infrastructure
+
+<!-- What infrastructural considerations need to be made for this project? If there are none, say so explicitly rather than deleting the section. -->
+
+TBD
+
+## Accessibility
+
+<!-- Are there specific accessibility concerns relevant to this project? Do you expect new UI elements that would need particular care to ensure they're implemented in an accessible way? Consider also low-spec device and slow internet accessibility, if relevant. -->
+
+- Follow the advisor of WCAG 2.2 to meet AA. Put special attention to the
+  [Perceivable section](https://www.w3.org/WAI/WCAG22/quickref/?currentsidebar=%23col_overview&levels=aaa&showtechniques=321#principle1).
```

Comment:
From the docs in the template:

> Are there specific accessibility concerns relevant to this project? **Do you expect new UI elements that would need particular care to ensure they're implemented in an accessible way?** Consider also low-spec device and slow internet accessibility, if relevant.

Is the component you envision for switching color themes one we already have in our component library, or a new control? Could it have any accessibility issues?

Reply:
> Is the component you envision for switching color themes one we already have in our component library, or a new control?

It already exist in the library.

> Could it have any accessibility issues?

If current checkbox, radio, and toggle components don't have accessibility issues, then everything is fine.

Reply:
Excellent. Can you mention that in the document?